config/src/main/java/com/typesafe/config/impl/ConfigIncludeKind.java

```diff
@@ -0,0 +1,5 @@
+package com.typesafe.config.impl;
+
+enum ConfigIncludeKind {
+    URL, FILE, CLASSPATH, HEURISTIC, STRING
```

Comment:
STRING is never used I think? just a string with no `url()` etc. around it would be heuristic "do what I mean" mode.